ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParsePayload.java

```diff
@@ -81,9 +81,10 @@ private static void loadSchema(String name) throws SchemaNotFoundException {
   }
 
   private static Schema getSchema(String name) throws SchemaNotFoundException {
-    final Schema schema = schemas.get(name);
+    Schema schema = schemas.get(name);
     if (schema == null) {
-      throw SchemaNotFoundException.forName(name);
+      loadSchema(name);
+      schema = schemas.get(name);
```

Comment:
Decoder running in Dataflow would always die on ParsePayload because it could never find the schemas.

The documentation for PTransform explains how PTransform is marked Serializable because it might be captured by a DoFn's closure, but its implementations of Serializability are no-ops. So this is actually documented behavior I hadn't previously understood.